.github/workflows/actions/deploy-terraform-infrastructure/action.yml

```diff
@@ -100,6 +100,10 @@ inputs:
   mithril_aggregator_cexplorer_pools_url:
     description: CExplorer url of the list of pools that is used by the mithril aggregator.
     required: false
+  mithril_aggregator_allow_unparsable_block:
+    description: Allow to log the unparsable block errors in the mithril aggregator.
```

Comment:
This is not clear enough: it should say that it logs the error instead of returning it (which means block parsing is not strictly enforced and should be used only in a test environment. 

Can you also make that this is also applied to all other comments/descriptions?

Reply:
Done ✅. Updated with the same as the one in the code.